org.dpsoftware.FireflyLuciferin.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": "org.dpsoftware.FireflyLuciferin",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "command": "/app/fireflyluciferin/bin/FireflyLuciferin",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=pulseaudio",
+    "--socket=x11",
+    "--socket=wayland",
+    "--device=all",
+    "--filesystem=home",
+    "--env=LD_LIBRARY_PATH=/app/lib",
+    "--env=GTK_THEME=Yaru:dark",
+    "--env=FLATPAK_ENV=1"
+  ],
+  "modules": [
+    "shared-modules/libayatana-appindicator/libayatana-appindicator-gtk3.json",
+    {
+      "name": "fireflyluciferin",
```

Comment:
the "main" app is always last. dependencies always come before.

Reply:
ok fixed it

Reply:
fixed